### DISCUSSION TITLE
Add ability to render next/image like regular <img> tag with intrinsic size

### DISCUSSION THREAD

**Discussion by MonstraG** [vercel/next.js#45902] [Ideas]
### Describe the feature you'd like to request

I display pages on a website, that mostly consist of one big (more than 1000x1000) image.

I don't want to re-build the website each time I add the image, so all of the images are hosted on CDN.

For users of mobile phones or tablets, when their screen size is smaller than the image size, I would like for them to have scrollbars so that they can see the details of the image (seeing the entirety of image is almost never important).

Best-fitting solution to these constrains seems to be to render the image in it's intrinsic resolution (and overflow).

Obviously in this case I don't care about layout shift (all of the page layout is actually either top or bottom bars, so it doesn't shift)

It seems that I cannot do that.

### Describe the solution you'd like

Either allow to unset `height` and `width` from NextImage, or add some other property that would allow me to do that.

### Describe alternatives you've considered

Right now I have to use plain `<img>` tag because it renders the images correctly.

Note, I did not create a discussion-question for this because it seems like nobody uses those.

**Comment by balazsorban44** [comment]
> Note, I did not create a discussion-question for this because it seems like nobody uses those.

Thanks, we do look at discussions and try to prioritize based on engagement.

Feature requests are converted to discussions as outlined in our contributing docs: https://github.com/vercel/next.js/blob/canary/contributing/repository/triaging.md#triaging

I'll pass this on to the team so we can have a look

**Comment by balazsorban44** [comment]
Could you provide a bit more detail about your problem with a code snippet?

Maybe the `fill` property satisfies your needs, as it does not require the size properties:

https://nextjs.org/docs/api-reference/next/image#fill
